package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test-changed": "lerna run test --since master",
     "test": "lerna run test",
-    "test-ci": "lerna run test-ci --concurrency 1",
+    "test-ci": "lerna run test --concurrency 1 -- --ci -i",
```

Comment:
Wouldn't it just be 
```suggestion
"test-ci": "lerna run test -i --ci --concurrency 1",
 ```

Reply:
Lerna only passes the args after "--" to the underlying command